Traceback (most recent call last):
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
"Albert Camus \"" + books["Albert Camus"] + "\" wird in der Coronakrise wieder viel gelesen."
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mKeyError[0m                                  Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_15232/3419878387.py[0m in [0;36m<module>[1;34m[0m
[1;32m----> 1[1;33m [1;34m"Albert Camus \""[0m [1;33m+[0m [0mbooks[0m[1;33m[[0m[1;34m"Albert Camus"[0m[1;33m][0m [1;33m+[0m [1;34m"\" wird in der Coronakrise wieder viel gelesen."[0m[1;33m[0m[1;33m[0m[0m
[0m
[1;31mKeyError[0m: 'Albert Camus'
KeyError: 'Albert Camus'

